modules/sdk-coin-kava/src/lib/keyPair.ts

```diff
@@ -0,0 +1,25 @@
+import { KeyPairOptions } from '@bitgo/sdk-core';
+import { pubkeyToAddress } from '@cosmjs/amino';
+
+import { CosmosKeyPair } from '@bitgo/abstract-cosmos';
+
+/**
+ * Kava keys and address management.
+ */
+export class KeyPair extends CosmosKeyPair {
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
+  /** @inheritdoc */
+  getAddress(): string {
+    const base64String = Buffer.from(this.getKeys().pub.slice(0, 66), 'hex').toString('base64');
+    return pubkeyToAddress(
+      {
+        type: 'tendermint/PubKeySecp256k1',
+        value: base64String,
+      },
+      'kava'
```

Comment:
Can we have this 'kava' stored somewhere in a variable? Like `const ADDRESS_SUFFIX` or whatver seems appropriate?